src/_includes/layouts/learning-resources-nav.njk

```diff
@@ -0,0 +1,70 @@
+<!-- Navigation -->
+<div class="border-r">
+    {% set navPages = collections.all | eleventyNavigation %}
+    {% macro renderNavListItem(entry) -%}
+        {% set isChildUrl = false %}
+        {% for child in entry.children %}
+            {% if child.url == page.url %}
+                {% set isChildUrl = true %}
+            {% endif %}
+        {% endfor %}
+
+        <li class="{% if entry.url==page.url and not isChildUrl %}active{% endif %}{% if entry.children.length %} has-children{% endif %}">
```

Comment:
Why can childURLs not be active?

Reply:
This is related to the classes of the core-nodes categories, because we don't have an index file and the name of the category redirects to the first child. If I didn't add this condition, when the first child was active, the parent remained active as well since they have the same URL